.changeset/modern-cougars-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+'tinacms': patch
+---
+
+enable / in search
```

Comment:
Can you provide more details for the change set?

Reply:
Done - updated changeset